src/docs/arch-column-row.dox

```diff
@@ -0,0 +1,93 @@
+/*! @arch_page arch-column-row Row Store & Column Store
+
+@section Definition
+
+A row oriented storage is an access method where all columns of a row are stored
+together, the data is stored by row, while a column oriented system stores its data tables by
+column. WiredTiger supports both and each management system is represented internally by a certain
+type of B-Tree data structure.
```

Comment:
I think the `the data is stored by row` phrase can be confusing -- column-store stores by row, too, doesn't it?

Reply:
I wanted the reader to think of the following:
A | B | C
D | E | F
is stored A | B | C | D | E | F for a row-store
and is stored A | D | B | E | C | F for a column-store

Reply:
Thanks for your detailed answer @ddanderson, I will remove the performance bits. As we discussed yesterday, we could feel we were not taking the right direction for this page.